src/include/distributed/metadata_sync.h

```diff
@@ -68,7 +68,7 @@ extern Oid GetAttributeTypeOid(Oid relationId, AttrNumber attnum);
 #define DELETE_ALL_NODES "TRUNCATE pg_dist_node CASCADE"
 #define DELETE_ALL_DISTRIBUTED_OBJECTS "TRUNCATE citus.pg_dist_object"
 #define REMOVE_ALL_CLUSTERED_TABLES_COMMAND \
```

Comment:
I think the downside of this approach is that if you follow below:
```SQL
select stop_metadata_sync_to_node('localhost', 9700);
alter sequence my_seq RENAME TO my_seq_2;
select start_metadata_sync_to_node('localhost', 9700);
```

We'd still hit #5303. But I think that is sort of okay for now given that stopping the metadata sync is not something we'd want to continue. 

I think the more appropriate way of fixing this would be to store the information about the sequences  on the coordinator when/if metadata is stopped. But, given that doesn't seem like real use-case, I went ahead with this simple approach.